core/src/main/java/com/github/taucher2003/appenders/core/github/IssueAppender.java

```diff
@@ -33,7 +33,6 @@
     protected List<String> labels = new ArrayList<>();
 
     public IssueAppender() {
-        labels.add("logger");
     }
```

Comment:
As the Constructor is now empty, it can be removed as it is no longer needed, if you want, you can do that.
Otherwise I will do it before merge.

Reply:
Removed the empty constructor.